cluster/gce/util.sh

```diff
@@ -386,7 +387,7 @@ function create-node-template {
       ${preemptible_minions} \
       $2 \
       --can-ip-forward \
-      --metadata-from-file "$3","$4" >&2; then
+      --metadata-from-file ${@: 3:$#} >&2; then
```

Comment:
From my simple local testing, this expansion doesn't seem to put commas between each argument, and gives `$3 $4` instead of `$3,$4` like we had before.